Makefile

```diff
@@ -103,13 +103,6 @@ addman: help2man man-db
 	help2man -Nh help -v version -n "configure and get information from your dokku installation" -o /usr/local/share/man/man1/dokku.1 dokku
 	mandb
 
-version:
-ifeq ($(DOKKU_VERSION),master)
-	git describe --tags > ~dokku/VERSION  2> /dev/null || echo '~${DOKKU_VERSION} ($(shell date -uIminutes))' > ~dokku/VERSION
-else
-	echo $(DOKKU_VERSION) > ~dokku/VERSION
```

Comment:
Can we keep this and have it output to `/var/lib/dokku/version`?

Reply:
Done!